contracts/FanDonation.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
nit: We're using MIT for the repo, so let's substitute agpl-3.0 with MIT for all of our authored contracts.